docs/source/slurmguide/slurmrunningjobs.md

```diff
@@ -1,97 +1,67 @@
-:::
-
-Some things Slurm assumes, like there is no overlap between different CPUs by default: tasks do not share CPUs with others running parallel. If overlap is needed, use the following Slurm flag:
-
-:::{code} bash
---overlap
-:::
-
-Also, set the environment variable `SLURM_OVERLAP=1` via
-:::{code} bash
-export SLURM_OVERLAP=1
-:::
-
-:::{important}
-Run `export SLURM_OVERLAP=1` prior to logging onto a compute node when using MPI interactively.
-:::
+You have two options when running tasks, run interactively via {ref}`using-srun` or by batch (passively) via {ref}`using-sbatch`.
 
 (using-srun)=
 ## Interactive Jobs: `srun` Command
-The `srun` command is used to submit an interactive job which runs a single task directly via the shell. This method is useful when you want to test a short computation or run an interactive session like a shell, Python, or an R terminal.
+The `srun` command is used to submit an interactive job which runs in a shell terminal. This method is useful when you want to test a short computation or run an interactive session like a shell, Python, or an R terminal.
 
-### Syntax: `srun`
+To start an `srun` session, the following syntex is used from the login node:
 
 :::{code} bash
 srun [options] [command]
 :::
 
-### Options and Usage: `srun`
-- **`n, --ntasks=<number>`**: specify the number of tasks
-- **`N, --nodes=<minnodes[-maxnodes]>`**: specify the number of nodes
-- **`J, --job-name=<jobname>`**: specify a name for the job
+As an example of an `srun` job running on 1 node, with 1 tasks:
 
 :::{code} bash
-srun -N 1 -n 1 --pty bash
+srun -N 1 -n 1 -p short --pty bash
 :::
 
-This command starts an interactive bash shell on one node with one task.
-
-### Examples using `srun`
+- **`n, --ntasks=<number>`**: specify the number of tasks
+- **`N, --nodes=<minnodes[-maxnodes]>`**: specify the number of nodes
+- **`p, --partition=<partition-name>`**: specify a partition for the job to run on
 
-The user needs to review the {ref}`hardware-overview` and {ref}`partition-names` to be familiar with the available hardware and partition limits on Discovery. This way, user can tailor the request to fit both the needs of the job and the limits of partitions. For example, if the user specifies `--partition=short` and `--time=01:00:00`, it will result in an error because the time specified exceeds the limit for that partition.
+To see all options for `srun`, please refer to [srun manual] from Schedmd.
 
-This simple `srun` example is to move to a *compute* node after you first log into the HPC:
+### Examples using `srun`
 
-:::{code} bash
-srun --pty /bin/bash
-:::
+You should review the {ref}`hardware-overview` and {ref}`partition-names` pages to be familiar with the available hardware and partition limits on Discovery. This way, you can tailor the request to fit both the needs of the job and the limits of partitions. 
```

Comment:
```suggestion
You can tailor your request to fit both the needs of the job and the partition limits if you're familiar with the available hardware and partitions on Discovery. 
```
Can we just add the "Hardware Overview" link to "available hardware," and "Partition Names" to the second "partition?"

Reply:
Also, maybe we can make the examples a bulleted list or something to make them easier to read?

Reply:
Are the examples the code blocks or something else? Thank you!!